screens/BooksScreen.tsx

```diff
@@ -15,19 +15,17 @@ import BooksList from '../components/BooksList';
 import { defaultStyles } from '../styles/default';
 
 export default function BooksScreen() {
-  const { data, loading, error, refetch } = useQuery<BookQuery>(GET_BOOKS, {
+  const [search, setSearch] = React.useState('');
+  const { data, loading, error } = useQuery<BookQuery>(GET_BOOKS, {
     notifyOnNetworkStatusChange: true,
-    variables: { search: '' },
+    variables: { search: search },
```

Comment:
Do you know this JS shorthand?

```suggestion
    variables: { search },
```

Reply:
@paour Yes, if the key and the value have the same name